photutils/psf/tests/test_photometry.py

```diff
@@ -498,10 +499,68 @@ def test_aperture_radius_value_error():
                                     fitshape=7)
 
     intab = Table(data=[[1], [1]], names=['x_0', 'y_0'])
-    with pytest.raises(ValueError) as err:
+    with pytest.warns(AstropyUserWarning):
         basic_phot(image=image, init_guesses=intab)
 
-    assert 'aperture_radius is None' in str(err.value)
+
+def tophatfinder(image):
```

Comment:
minor and debatable (i.e., if you disagree I won't push it), but I would say it makes more sense to put this inside the test function below instead of at the module-level, since it's specific to that one test.

Reply:
That does make more sense; I moved the function inside the corresponding test.